src/app/ui/styles/auto-complete-popup-styles.js

```diff
@@ -40,11 +38,9 @@ var css = csjs`
     margin-left      : 20%;
     margin-bottom    : 32px;
     bottom           : 0px;
-    background-color : ${styles.colors.black};
```

Comment:
this modal is now appearing a the top of the screen - not above the terminal like it used to.

Reply:
i've disabled the autocomplete for now... 
(cause it will need a bit more work)